libqtile/backend/x11/core.py

```diff
@@ -449,27 +461,7 @@ def display_name(self) -> str:
         return self._display_name
 
     def update_client_lists(self) -> None:
-        """Updates the _NET_CLIENT_LIST and _NET_CLIENT_LIST_STACKING properties
-
-        This is needed for third party tasklists and drag and drop of tabs in
-        chrome
-        """
-        assert self.qtile
-        # Regular top-level managed windows, i.e. excluding Static, Internal and Systray Icons
-        wids = [wid for wid, c in self.qtile.windows_map.items() if isinstance(c, window.Window)]
-        self._root.set_property("_NET_CLIENT_LIST", wids)
-
-        # We rely on the stacking order from the X server
-        stacked_wids = []
-        for wid in self._root.query_tree():
-            win = self.qtile.windows_map.get(wid)
-            if not win:
-                continue
-
-            if isinstance(win, window.Window) and win.group:
-                stacked_wids.append(wid)
-
-        self._root.set_property("_NET_CLIENT_LIST_STACKING", stacked_wids)
+        self.zmanager.update_client_lists()
```

Comment:
Can the X11 Core inherit from Zmanager instead of having this proxy / reference loop between zmanager and core?

Reply:
Yes. That's a nice idea.